### ISSUE TITLE
PopoverMenu component should have tooltip

### ISSUE THREAD

**Issue by kaseyvee** [rustic-ai/ui-components#158]
**Is your feature request related to a problem? Please describe.**
It's recommended that buttons without text should have tooltips for clarity and accessibility.

**Describe the solution you'd like**
A tooltip should be added to the PopoverMenu component for icon buttons (buttons with no text).
